foundations/javascript_basics/fundamentals-2.md

```diff
@@ -48,11 +48,17 @@ To give you a good bit of practice, we have created some repl.it exercises for y
 
 Be sure to do the lessons in order presented here. Pressing "run" at the top will run the code. Read all directions, watch the terminal, and read all the errors. Don't forget to use 'console.log' extensively.
 
+To get started, create a free repl account and click "Fork" in order to have access to the exercises. 
+Note: Feel free to browse the files on the left column in order to gain familarity with it. 
+
 - [Exercise 1](https://repl.it/@I3uckwheat/troubleshooting#troubleshooting.js)
+  *In this excercise, you will be working out of the file called trobuleshooting.js
 - [Exercise 2](https://repl.it/@I3uckwheat/enter-a-number#script.js)
+  *You will be working out of script.js 
 - [Exercise 3](https://repl.it/@I3uckwheat/lets-do-some-math#math.js)
+  *You will be working out of math.js  
 - [Exercise 4](https://repl.it/@I3uckwheat/direction-follow#follow.js)
-
+  *You will be working out of follow.js
```

Comment:
Not quite, you'll need a space after the `*`.

Feel free to play around with our new preview tool as well: https://www.theodinproject.com/lessons/preview

Reply:
I was blindly guessing at what I was doing, but that preview tool is VERY helpful. Thanks alot Briggs !

Reply:
Thank YOU for doing this :)